ProjectBank/Server/Model/Extensions.cs

```diff
@@ -0,0 +1,16 @@
+namespace ProjectBank.Server.Model;
+
+public static class Extensions
+ {
+//     public static IActionResult ToActionResult(this Status status) => status switch
+//     {
+//         Updated => new NoContentResult(),
+//         Deleted => new NoContentResult(),
+//         NotFound => new NotFoundResult(),
+//         Conflict => new ConflictResult(),
+//         _ => throw new NotSupportedException($"{status} not supported")
+//     };
+
+//     public static ActionResult<T> ToActionResult<T>(this Option<T> option) where T : class
+//         => option.IsSome ? option.Value : new NotFoundResult();
```

Comment:
Hvorfor er det her ud-kommenteret @Werenfeldt? :D Skal vi ikke bruge det, for så tænker jeg at fjerne det inden Main.